src/coreclr/jit/loopcloning.cpp

```diff
@@ -45,6 +45,22 @@ void ArrIndex::PrintBoundsCheckNodes(unsigned dim /* = -1 */)
     }
 }
 
+//--------------------------------------------------------------------------------------------------
+// Print: debug print an SpanIndex struct in form: `V01[V02]`.
+//
+void SpanIndex::Print()
+{
+    printf("V%02d[V%02d]", lenLcl, indLcl);
```

Comment:
This seems a bit odd, as it indicates that `lenLcl` is an "array", but it's actually just the length of the array (in the span). Maybe use something like `Span<V%02d>[V%02d]` instead? Basically, something to make it obvious that `lenLcl` is not an array.

Reply:
yep, copy-paste from ArrIndex 🙂 I'll fix the comment in a follow up (I think I found a few missing opportunities for LC)